README.md

```diff
@@ -1758,6 +1757,30 @@ List of supported GmailFilter resources:
 *   `filter`
     * `gmailfilter_filter`
 
+### Use with Kibana
+
+Check the documentation of [go-kibana](https://github.com/ewilde/go-kibana) for which env vars to set
+
+Example:
+
+```
+export KIBANA_USERNAME=user
+export KIBANA_PASSWORD=password
+export KIBANA_TYPE=KibanaVanilla
```

Comment:
Please other options(logs.io?)

Reply:
Do you mean to prefer an example for Logz.io instead of vanilla Kibana? There is a mention and link to `go-kibana` where it list all the env vars that are supported.